CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -169,7 +160,7 @@ class CameraFragment : Fragment() {
         return fragmentCameraBinding.root
     }
 
-    private fun setGalleryThumbnail(uri: Uri) {
+    private fun setGalleryThumbnail(filename: String) {
         // Run the operations in the view's thread
         cameraUiContainerBinding?.photoViewButton?.let { photoViewButton ->
             photoViewButton.post {
```

Comment:
nit: since this should all be main thread safe now -- we can probably remove the `.post` call but just double check this is true

Reply:
Removing this causes a Glide error, saying it's not called on the UI thread. I could alternatively put this in a `launch` block, but I chose to leave it as is. Let me know if you'd like a followup change for this.